api/api.go

```diff
@@ -243,7 +254,7 @@ func getOrganization(ctx context.Context, logger *logger.Logger, organizationNam
 	err := graphQLclient.Run(ctx, request, &response)
 
 	if err != nil || response.Organization.ID == "" {
-		err = errors.Wrap(err, fmt.Sprintf("Unable to find organization %s of vcs-type %s", organizationName, organizationVcs))
+		err = fmt.Errorf("Unable to find organization %s of vcs-type %s", organizationName, organizationVcs)
```

Comment:
This change is destructive - the original error is lost. Let's suppose that error is a network failure, we want the error message to include this information, so that the user can take action.

Reply:
Fair enough. This will need to become an if-else statement, then. 

From what @johnswanson and I could tell, when the error is `nil` (like when the response is empty), if you wrap it with a string the end result is still `nil` (so the subsequent `err != nil` checks don't behave as-desired).

Do you know if there's a better way to handle this?

Reply:
@marcomorain @hannahhenderson The change isn't destructive.  These two lines have equivalent return values.  `fmt.Errorf` returns an `error` object with string interpolation:

https://golang.org/pkg/fmt/#Errorf

See also: https://www.reddit.com/r/golang/comments/6ffrie/fmterrorf_or_errorsnew/